frontend/src/views/v-authorship.vue

```diff
@@ -93,14 +93,14 @@
 
   .files(v-if="isLoaded")
     .empty(v-if="files.length === 0") nothing to see here :(
-    template(v-for="(file, i) in selectedFiles")
+    span(v-for="(file, i) in selectedFiles")
       .file(v-bind:key="file.path")
         .title
           span.path(v-on:click="toggleFileActiveProperty(file)")
             .tooltip
-              font-awesome-icon(icon="caret-down", fixed-width, v-show="file.active")
+              font-awesome-icon(v-if="file.active", icon="caret-down", fixed-width)
               span.tooltip-text(v-show="file.active") Click to hide file details
-              font-awesome-icon(icon="caret-right", fixed-width, v-show="!file.active")
+              font-awesome-icon(v-if="!file.active", icon="caret-right", fixed-width)
```

Comment:
@gerhean I know there is a slight difference between `v-show` and `v-if` but I'm not sure why it makes such a big difference here. Originally when it is `v-show`, there will be an error and the toggling does not work as expected.

Reply:
`v-show` prerenders the content and toggles visibility off, `v-if` only renders content if condition is true.

I'm just guessing, but this might be due to the `font-awesome-icon` not being loaded before the file is loaded (I would look at maybe the import order of various files), which causes the error.